tests/Nox.Types.Tests/NoxCurrencyCode3Tests.cs

```diff
@@ -0,0 +1,181 @@
+    [InlineData("BMD")]
+    [InlineData("BND")]
+    [InlineData("BOB")]
+    [InlineData("BRL")]
+    [InlineData("BSD")]
+    [InlineData("BTC")]
+    [InlineData("BTN")]
+    [InlineData("BWP")]
+    [InlineData("BYR")]
+    [InlineData("BYN")]
+    [InlineData("BZD")]
+    [InlineData("CAD")]
+    [InlineData("CDF")]
+    [InlineData("CHF")]
+    [InlineData("CLP")]
+    [InlineData("CNY")]
+    [InlineData("COP")]
+    [InlineData("CRC")]
+    [InlineData("CUC")]
+    [InlineData("CUP")]
+    [InlineData("CVE")]
+    [InlineData("CZK")]
+    [InlineData("DJF")]
+    [InlineData("DKK")]
+    [InlineData("DOP")]
+    [InlineData("DZD")]
+    [InlineData("EGP")]
+    [InlineData("ERN")]
+    [InlineData("ETB")]
+    [InlineData("EUR")]
+    [InlineData("FJD")]
+    [InlineData("FKP")]
+    [InlineData("GBP")]
+    [InlineData("GEL")]
+    [InlineData("GHS")]
+    [InlineData("GIP")]
+    [InlineData("GMD")]
+    [InlineData("GNF")]
+    [InlineData("GTQ")]
+    [InlineData("GYD")]
+    [InlineData("HKD")]
+    [InlineData("HNL")]
+    [InlineData("HRK")]
+    [InlineData("HTG")]
+    [InlineData("HUF")]
+    [InlineData("IDR")]
+    [InlineData("ILS")]
+    [InlineData("INR")]
+    [InlineData("IQD")]
+    [InlineData("IRR")]
+    [InlineData("ISK")]
+    [InlineData("JMD")]
+    [InlineData("JOD")]
+    [InlineData("JPY")]
+    [InlineData("KES")]
+    [InlineData("KGS")]
+    [InlineData("KHR")]
+    [InlineData("KMF")]
+    [InlineData("KPW")]
+    [InlineData("KRW")]
+    [InlineData("KWD")]
+    [InlineData("KYD")]
+    [InlineData("KZT")]
+    [InlineData("LAK")]
+    [InlineData("LBP")]
+    [InlineData("LKR")]
+    [InlineData("LRD")]
+    [InlineData("LSL")]
+    [InlineData("LYD")]
+    [InlineData("MAD")]
+    [InlineData("MDL")]
+    [InlineData("MGA")]
+    [InlineData("MKD")]
+    [InlineData("MMK")]
+    [InlineData("MNT")]
+    [InlineData("MOP")]
+    [InlineData("MRO")]
+    [InlineData("MTL")]
+    [InlineData("MUR")]
+    [InlineData("MVR")]
+    [InlineData("MWK")]
+    [InlineData("MXN")]
+    [InlineData("MYR")]
+    [InlineData("MZN")]
+    [InlineData("NAD")]
+    [InlineData("NGN")]
+    [InlineData("NIO")]
+    [InlineData("NOK")]
+    [InlineData("NPR")]
+    [InlineData("NZD")]
+    [InlineData("OMR")]
+    [InlineData("PAB")]
+    [InlineData("PEN")]
+    [InlineData("PGK")]
+    [InlineData("PHP")]
+    [InlineData("PKR")]
+    [InlineData("PLN")]
+    [InlineData("PYG")]
+    [InlineData("QAR")]
+    [InlineData("RON")]
+    [InlineData("RSD")]
+    [InlineData("RUB")]
+    [InlineData("RWF")]
+    [InlineData("SAR")]
+    [InlineData("SBD")]
+    [InlineData("SCR")]
+    [InlineData("SDD")]
+    [InlineData("SDG")]
+    [InlineData("SEK")]
+    [InlineData("SGD")]
+    [InlineData("SHP")]
+    [InlineData("SLL")]
+    [InlineData("SOS")]
+    [InlineData("SRD")]
+    [InlineData("STD")]
+    [Theory]
+    public void Nox_CurrencyCode3_Constructor_ReturnsSameValue_AllCurrencies(string currencyCode3String)
```

Comment:
Can you please remove Nox prefix from all tests and look to this one https://regusit.atlassian.net/wiki/spaces/NOX/pages/3153461266/Coding+Conventions+Styles+and+Guidelines we are approving this.

See also  https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#best-practices , having tests names with Method, Scenario,Expected Result

```suggestion
    public void CurrencyCode3_Constructor_ReturnsSameValue_AllCurrencies(string currencyCode3String)
```